RecipeApp/Model/Recipe/Recipe.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Recipe.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-11-21.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import Foundation
+
+struct Recipe: Codable {
+    var meals: [Meal]
+}
+
+struct Meal: Codable {
+    var mealName: String
```

Comment:
This is VERY picky (!) but since this struct is called Meal, the fact that it's a mealName or mealImage is kind of implied, isn't it? Just name or image would be fine since you'd access it like "let mealName = meal.name", etc.

Not saying you should change it - just being picky!! Some places you will work at will catch those things, too - seriously!